EVPurchaseProgram/src/main/java/ro/siit/evprogram/FilterCars.java

```diff
@@ -0,0 +1,20 @@
+package ro.siit.evprogram;
+
+public class FilterCars {
+
+    /**
+     * Created method in order to filter the list based on the fast charging  and stock criteria
+     *
+     * @param ev
+     * @return
+     */
+
+    public ElectricVehicle[] filter(ElectricVehicle[] ev) {
+        for (int i = 0; i < ev.length - 1; i++) {
+            if ((ev[i].isFastCharging() == true) && (ev[i].isStock() == true)) {
```

Comment:
There should be 2 filtering modes: 
- filter by is fast-charging
- filter by is in stock